linodecli/cli.py

```diff
@@ -133,6 +133,7 @@ def _parse_properties(self, node, prefix=[]):
                     '.'.join(prefix+[name]),
                     info.get('x-linode-filterable') or False,
                     info.get('x-linode-cli-display') or False,
+                    info.get('type') or 'string',
```

Comment:
I think this is what is currently happening in a way. So this line preserves the default of `string` as a fallback.